src/message_replies/reportChannelPoints.js

```diff
@@ -33,8 +33,17 @@ const reportChannelPoints = async (interaction) => {
   const userListHTML = leaderBoardBody(channelName, caller, arrOfValidUsers);
   const body = generateBody(userListHTML, styles);
 
-  const img = await imgFromHtmlGenerator(body);
-  interaction.reply({ files: [{ attachment: img }] });
+  try {
+    const img = await imgFromHtmlGenerator(body, interaction);
+    if (!(img instanceof Error)) {
+      interaction.reply({ files: [{ attachment: img }] });
```

Comment:
I understand why the `if (instanceof )` but I'm not sure if it would be enough in this case. In fact, AFAIK there's no reason why it should fall in the catch right after this part. If you think that it would make sense that, if is instance of Error, to throw an error

Reply:
@Claudi0-V 

Im not sure that I'm understanding your question. The way this is intended to write is that it calls `imageFromHtmlGenerator` and if that method returns back data, it will send it to discord. If it throws an error, it will be caught in the catch and the logged. At that point an error message will be sent to Discord. 

The `instanceOf Error` is because if it's not an error, we can send the data but if it is an error, we dont want to send it, we just want to be done at that point. I think the way that it was written before, if that method returned back an error, it would send it to Discord.